talent/contract.md

```diff
@@ -1,19 +1,25 @@
 # Work contract
 This is the place where you will find all the information regarding your work contract.<br>
-<br>
-You can find here the <a href="">English</a> and <a href="">Spanish</a> version of our contract.
+
+You can find here the [English](https://drive.google.com/file/d/1LMKOmpTeLQZlEEKK7CkiJW8iC7HC-I0O/view?usp=sharing) and [Spanish](https://drive.google.com/file/d/1bDNPwht2GVnz8p3AfRDDp_mOxLFgetXr/view?usp=sharing)  version of our contract.
+
 ### Main clauses
-1. Description of your position and where you will work. Even though it says you'll work from our Madrid office, we have a <a href="">remote working policy</a> that allows you to work from home as well. <b>link missing</b>
+1. Description of your position and where you will work. Even though it says you'll work from our Madrid office, we have a remote working policy that allows you to work from home as well. We have an additional clause explaining it. 
```

Comment:
I'd add:
>You can take a look at `[our remote work policy](flexible_holidays_working_schedule_remote_work.md#working-remotely)`

Reply:
@dpordomingo  I have a question. It's true that the links in markdown are written the way you mentioned. But is it wrong to add links in html tags?

Reply:
I have no strong opinion on link format... but let's compare:
`<a href="flexible_holidays_working_schedule_remote_work.md#working-remotely">our remote work policy</a>`
vs
`[our remote work policy](flexible_holidays_working_schedule_remote_work.md#working-remotely)`

It's shorter in markdown, but it's only my opinion.

Anyhow, my suggestion was about adding a link to the remote policy just after saying
>_We have an additional clause explaining it. _

whatever you do, is ok for me :smile:

Reply:
ok ok! I just didn't know if there is any convention :)